kieserver/image.yaml

```diff
@@ -91,6 +91,12 @@ envs:
     - name: "AUTO_CONFIGURE_EJB_TIMER"
       example: "false"
       description: "Set if the EJB timer datasources should be auto configured, if false, user will need to define the default and the ejb datasources using the DATASOURCES or DB_SERVICE_PREFIX_MAPPING environment variables manually. Default is true"
+    - name: "KIE_EJB_TIMER_TX"
+      example: "false"
+      description: "Set if the EJB timer is in use, if true enables this property enables the transactions for EJB timer. Defaults to true"
+    - name: "KIE_EJB_TIMER_LOCAL_CACHE"
+      example: "true"
+      description: "Set if the EJB timer is in use, if false this property turns off the EJB Timers local cache. Defaults to false"
```

Comment:
We want to be consistent with describing what "true" does, I think. Describing what "false" does can confuse the reader. So, a rewrite:
"If the EJB timer is in use, setting this parameter to true enables the local cache for the EJB timer. The default is false"

Reply:
THanks Misha, applied.